packages/nodejs/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+    "extends": "../../tsconfig-base.json",
+    "include": ["src/**/*"],
+    "exclude": ["node_modules", "grpc/**/*"],
+    "rerefences": [
+        { "path": "../common"}
+    ],
+    "compilerOptions": {
+        "rootDir": "src",
+        "outDir": "./lib/src"
```

Similar to web package, would it be nicer with

```suggestion
        "outDir": "./lib"
```

---

I guess we should be alert doing this, it might break something :upside_down_face:

---

I guess it would. But we're still in the beta versions, aren't we?